playground/DEPLOY.md

```diff
@@ -0,0 +1,95 @@
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+## Manual App Install
+To install Playground applications into existed GCP account you'll need to follow several steps, in this document just simple set of commands will be provided that will guide you over install process.
+
+Before starting that process, you will need to create GCP Artifact Registry instance to store there docker files that would be deployed. Guide to install that registry could be found in [that](https://github.com/apache/beam/blob/master/playground/terraform/README.md) document.
+### Common steps
+First of all you will need to get service account key in json format, that can be done by following [that](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) document. Let's assume that you saved that key into file named key.json.
+Next step is to authentificate your account in GCP docker registry, this can be done by running command:
+```bash
+$ cat /root/dt-key.json | docker login -u _json_key --password-stdin REGISTRY_NAME
+```
+you will need to replace `REGISTRY_NAME` to actual registry address that can be found in Google Cloud console of via gcloud tool.
```

Comment:
Having the doc link (https://cloud.google.com/artifact-registry/docs/manage-repos#view) or example here would be helpful (either the gcloud command like `gcloud artifacts repositories list` or  something like `us-central1-docker.pkg.dev/my-project/playground-repository`).

I think it would also be better to have this link to the actual guides from GCP documentation to create AR repositories like https://cloud.google.com/artifact-registry/docs/manage-repos